internal/cli/command/execution/execution.go

```diff
@@ -26,7 +28,9 @@ func NewCmd(octaviusDaemon daemon.Client) *cobra.Command {
 			}
 			client := &client.GrpcClient{}
 			response, err := octaviusDaemon.ExecuteJob(jobName, jobData, client)
-			logger.Error(err, response.Status)
+			if err != nil {
+				log.Error(err, response.Status)
```

Comment:
in `log.Error()` if `err != nil`, we get the error logged and if the `err == nil`, we get the msg passed in the second param logged as `info`. 

Can we skip the `err!=nil` check here as `response.Status` will never got logged out in this case

Reply:
that's a bad design and not a best practice. see https://github.com/gopay-bootcamp/octavius/issues/40.

you need to handle your error. not let the `log` decide what to do with your error

Reply:
```
func (l *Logger) Err(err error) *Event {
	if err != nil {
		return l.Error().Err(err)
	}

	return l.Info()
}
```
This is the code inside `zerlog.Err()` method. There is a `err!=nil` check already inside. That is the reason i was saying to do something like that. 😅

Reply:
Ah this one, I will remove this.

Reply:
Yes, but we not implement err checking like that in Octavius. I believe zerolog didn't create this to let us give error handling to log library.